### PR TITLE
data: sriharshamajeti

### PR DESCRIPTION
**Pull request by sriharshamajeti**

### CODE REVIEW

File: data/sriharshamajeti.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Sara Jaoude",
```
Comment (ChinmayMhatre) [inline data/sriharshamajeti.json:2]: Please change it to your name.

File: data/sriharshamajeti.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
```
Comment (ChinmayMhatre) [inline data/sriharshamajeti.json:5]: I think this is the bio of sara, you might want to change this

File: data/sriharshamajeti.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/sri-harsha-majeti/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/sriharshamajeti",
+      "icon": "FaGitHub"
```
Comment (ChinmayMhatre) [inline data/sriharshamajeti.json:26]:
```suggestion
      "icon": "FaGithub"
```
The icon name casing is wrong

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

**Review by Tejender1521 — CHANGES_REQUESTED** [review]
Replace name with your name and use appropriate Icon names as mentioned in docs.

**Pull request closed without merging**